src/main/java/cc/hyperium/mods/chromahud/ElementRenderer.java

```diff
@@ -128,16 +126,21 @@ public static void draw(int x, double y, List<String> list) {
     public static void drawChromaString(String text, int xIn, int y) {
         FontRenderer renderer = Minecraft.getMinecraft().fontRendererObj;
         int x = xIn;
-        for (char c : text.toCharArray()) {
-            long dif = (x * 10) - (y * 10);
-            if (current.isStaticChroma())
-                dif = 0;
-            long l = System.currentTimeMillis() - dif;
-            float ff = current.isStaticChroma() ? 1000.0F : 2000.0F;
-            int i = Color.HSBtoRGB((float) (l % (int) ff) / ff, 0.8F, 0.8F);
-            String tmp = String.valueOf(c);
-            renderer.drawString(tmp, (float) ((double) x / getCurrentScale()), (float) ((double) y / getCurrentScale()), i, current.isShadow());
-            x += (double) renderer.getCharWidth(c) * getCurrentScale();
+        try {
+            for (char c : text.toCharArray()) {
+                long dif = (x * 10) - (y * 10);
+                if (current.isStaticChroma()) {
+                    dif = 0;
+                }
+                long l = System.currentTimeMillis() - dif;
+                float ff = current.isStaticChroma() ? 1000.0F : 2000.0F;
+                int i = Color.HSBtoRGB((float) (l % (int) ff) / ff, 0.8F, 0.8F);
+                String tmp = String.valueOf(c);
+                renderer.drawString(tmp, (float) ((double) x / getCurrentScale()), (float) ((double) y / getCurrentScale()), i, current.isShadow());
+                x += (double) renderer.getCharWidth(c) * getCurrentScale();
+            }
+        } catch (NullPointerException null_pointer) {
+            null_pointer.printStackTrace();
```

Comment:
Could you change the exception name so that it uses _lowerCamelCase_, as opposed to the dashes, so as to follow convention.